tests/unit/lms/services/blackboard_api/_basic_test.py

```diff
@@ -163,23 +105,12 @@ def test_request_raises_HTTPError_if_the_HTTP_request_fails(
             basic_client.request("GET", "foo/bar/")
 
     @pytest.fixture
-    def basic_client(self, http_service, oauth_http_service, oauth2_token_service):
+    def basic_client(self, http_service, oauth_http_service):
         return BasicClient(
             blackboard_host="blackboard.example.com",
             client_id=sentinel.client_id,
             client_secret=sentinel.client_secret,
             redirect_uri=sentinel.redirect_uri,
             http_service=http_service,
             oauth_http_service=oauth_http_service,
-            oauth2_token_service=oauth2_token_service,
```

Comment:
It's good we've tucked away one element, but it still sticks out to me that `client_id`, `client_secret` and `redirect_uri` which are all OAuth related aren't encapsulated inside the service in some way too.